Makefile

```diff
@@ -139,7 +158,7 @@ fluentd:
 image-cleanup-test-images:
 	cd misc/image-cleanup-test-images; $(MAKE) $(MFLAGS)
 
-get-deps:
+get-deps: get-cni-sources
```

Comment:
Don't we typically use this even for setting up a dev environment ?
The new agent would require the cni-plugin packaging.